packages/server/src/models/social-profile.dto.ts

```diff
@@ -1,12 +1,14 @@
-import { IsString } from 'class-validator';
+import { IsString, IsOptional } from 'class-validator';
 
 export default class SocialProfileDto {
   @IsString()
   public pictureUrl: string;
 
+  @IsOptional()
   @IsString()
   public facebookId?: string;
```

Comment:
Just curious, what does the question mark affect in this case?

Reply:
Type checking in code